### DISCUSSION TITLE
Effective Dart?

### DISCUSSION THREAD

**Discussion by siew24** [XMUMY/XMUX#3] [Q&A]
A commit that enabled Effective Dart was pushed last October but should we just ignore the warnings?

**Comment by ctrysbita** [comment]
The linter rules are best practice of writing new code. Old warnings can be ignored since the old code hasn't been migrated yet.

**Reply by siew24** [reply]
In a sense, what do you mean by "migrating" the code?

**Reply by ctrysbita** [reply]
Means update the code to follow the latest rule.

**Reply by siew24** [reply]
I can handle that if you want

**Reply by siew24** [reply]
Nevermind that, what Flutter version are you using?

**Reply by ctrysbita** [reply]
I think the last version is 1.22
